src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -169,6 +170,21 @@ && isRightShooterUpToSpeed()
         || ignoreFlywheelSpeed);
   }
 
+  /**
+   * @return The current position of the shooter in rotations
+   */
+  public double getShooterPosition() {
+    return pivotMotor.getPosition().getValueAsDouble();
+  }
+
+  /**
+   * @return If the shooter position is within tolerance of desired position
+   */
+  public boolean isShooterAtPosition(Measure<Angle> position) {
+    return (Math.abs(getShooterPosition() - position.in(Units.Rotations)) < constShooter.AT_POSITION_TOLERANCE
+        .in(Units.Rotations));
+  }
```

Comment:
After getShooterPosition is returned as a rotations measure
```suggestion
    return (Math.abs(getShooterPosition().minus(position).in(Units.Rotations)) < constShooter.AT_POSITION_TOLERANCE
```

Reply:
![image](https://github.com/user-attachments/assets/7997ef6b-1d86-41fb-b77f-cbbdda0f0a21)
Would this also work? It keeps the values as Measure<Angle> and the .lte claims that it compares the magnitude of the two values 🤷‍♀️

Reply:
I am unsure if it does an absolute value or not. -100 is always less than .1
Feel free to test it out and give it a try though. If it works, that would make things much easier. It does say magnitude so it may work as magnitude is always positive.